prometheus_hardware_exporter/collectors/redfish.py

```diff
@@ -38,28 +63,62 @@ def get_sensor_data(self, host: str, username: str, password: str) -> Dict[str,
             output[name] = sensors
         return output
 
-    def _get_sensor_data(self, host: str, username: str, password: str) -> Optional[List[Any]]:
-        """Return sensor if sensor exists else None."""
+    def _get_sensor_data(
```

Comment:
two methods with same name?

Reply:
Private and public?

Reply:
Yep, `_get_sensor_data` is internal to the class while `get_sensor_data` actually returns data to the outside.